src/ast/mod.rs

```diff
@@ -3401,6 +3401,13 @@ pub enum Statement {
         partitioned: Option<Vec<Expr>>,
         table_format: Option<HiveLoadDataFormat>,
     },
+    /// ```sql
+    /// Rename TABLE tbl_name TO new_tbl_name[, tbl_name2 TO new_tbl_name2] ...
+    /// ```
+    /// renames one or more tables
+    ///
+    /// See Mysql <https://dev.mysql.com/doc/refman/9.1/en/rename-table.html>
+    RenameTable { operations: Vec<RenameObjectDef> },
```

Comment:
Could we do something like?
```rust
struct RenameTable {
    old_table: ObjectName,
    new_table: ObjectName
}
Statement::RenameTable(RenameTable)
```
using a dedicated struct makes it easier to extend (e.g with spans later on) and for downstream crates to work with the statement

Reply:
Good idea! Have Done.